modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -79,22 +79,56 @@ $ curl -L $ISO_URL -o rhcos-live.iso
 
 . Use the {op-system} ISO and the hosted `worker.ign` file to install the worker node:
 
-.. Boot the target host using the {op-system} ISO and your preferred method of installation.
+.. Boot the target host with the {op-system} ISO and your preferred method of installation.
 
-.. When the target host has booted from the {op-system} ISO, open a console on the target host, and run the following `coreos-installer` command to install the system:
+.. When the target host has booted from the {op-system} ISO, open a console on the target host.
+
+.. If your local network does not have DHCP enabled, configure the static IP of the worker node before running the {op-system} installation. For example, run the following commands on the target host console:
+
+... Configure the worker host network connection with a static IP:
 +
 [source,terminal]
 ----
-$ coreos-installer install --ignition-url=<hosted_worker_ign_file> <hard_disk>
+$ nmcli con mod <network_interface> ipv4.method manual /
+ipv4.addresses <static_ip> ipv4.gateway <network_gateway> ipv4.dns <dns_server>
+----
++
+where:
++
+--
+<static_ip>:: Is the host static IP address and CIDR, for example, `10.1.101.50/24`
+<network_gateway>:: Is the network gateway, for example, `10.1.101.1`
+--
+
+... Activate the modified network interface:
++
+[source,terminal]
+----
+$ nmcli con up <network_interface>
+----
+
+... Run the `coreos-installer` passing in the static IP and DNS details:
++
+[source,terminal]
+----
+$ sudo coreos-installer install --copy-network /
+--ignition-url=<hosted_worker_ign_file> <hard_disk> --insecure-ignition
 ----
 +
 where:
 +
 --
-<hosted_worker_ign_file>:: is the locally accessible URL for the hosted `worker.ign` file, for example, `http://webserver.example.com/worker.ign`
-<hard_disk>:: is hard disk where you install {op-system}, for example, `/dev/sda`.
+<hosted_worker_ign_file>:: Is the locally accessible URL for the hosted `worker.ign` file, for example, `http://webserver.example.com/worker.ign`
+<hard_disk>:: Is the hard disk where you install {op-system}, for example, `/dev/sda`
```

Comment:
We miss the instruction to set the hostname.

a.  Create another ignition file to include instruction to populate /etc/hostname and to point to the file above. In our  example:

{
    "ignition": {
        "version": "3.2.0",
        "config": {
            "merge": [
                {
                    "source": "http://webserver.example.redhat.com/worker.ign"
                }
            ]
        }
    },
    "storage": {
        "files": [
            {
                "path": "/etc/hostname",
                "contents": { "source": "data:,newhostname.example.redhat.com" },
                "mode": 420,
                "overwrite": true,
                "path": "/etc/hostname"
           }
        ]
    }
}

b. Upload the new ignition file to be reachable via web (http://webserver.example.redhat.com/NEW-worker.ign)
c.  sudo coreos-installer install --copy-network --ignition-url http://webserver.example.redhat.com/NEW-worker.ign /dev/sda --insecure-ignition

Reply:
Oops. Updated

Reply:
PTAL

Reply:
Looks good.
Could you please show me the generated doc - I want to make sure the steps are clear :)

Reply:
Surely. Here you go: https://54296--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes.html#sno-adding-worker-nodes-to-single-node-clusters-manually_add-workers

Reply:
So we can leave as is or change:
"Create a new ignition file new-worker.ign that includes the information that the coreos-installer program uses to populate /etc/hostname on the new worker host." 
to something like
"Create a new ignition file new-worker.ign that includes a reference to the original worker.ign and an additional instruction that the coreos-installer program uses to populate /etc/hostname on the new worker host."

But if you feel it's clear enough without it - it's ok.

Reply:
Made this change as suggested.